hs/src/Reach/APICut.hs

```diff
@@ -338,8 +338,8 @@ apc hc eWho = \case
   p -> return p
 
 apicut :: PLProg -> IO PLProg
-apicut (PLProg at plo dli dex epps cp) = do
+apicut (PLProg at plo dli dex epps cp ssm) = do
```

Comment:
I have a preference to put "extra" stuff before `epps`. Probably `PLProg` should become a record soon and these should be given nice names